doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -174,44 +188,48 @@ and push the changes to your fork
 GitHub pull request
 -------------------
 
-The next step is to open a GitHub pull request. The purpose of this step is
-to make your bugfix available to the community of Ceph developers.  They
-will review it and may do additional testing on it.
+The next step is to open a GitHub pull request (_PR_). This makes your bugfix
+visible to the community of Ceph contributors.  They will review it and may
+perform additional testing and / or request changes.
 
-In short, this is the point where you "go public" with your modifications.
-Psychologically, you should be prepared to receive suggestions and
-constructive criticism. Don't worry! In our experience, the Ceph project is
-a friendly place!
+This is the point where you "go public" with your modifications.  Be prepared
+to receive suggestions and constructive criticism in the form of comments
+within the PR. Don't worry!  The Ceph project is a friendly place!
 
-If you are uncertain how to use pull requests, you may read
+If you are uncertain how to create and manage pull requests, you may read
 `this GitHub pull request tutorial`_.
 
 .. _`this GitHub pull request tutorial`:
    https://help.github.com/articles/using-pull-requests/
 
-For some ideas on what constitutes a "good" pull request, see
+For ideas on what constitutes a "good" pull request, see
 the `Git Commit Good Practice`_ article at the `OpenStack Project Wiki`_.
 
 .. _`Git Commit Good Practice`: https://wiki.openstack.org/wiki/GitCommitMessages
 .. _`OpenStack Project Wiki`: https://wiki.openstack.org/wiki/Main_Page
 
+and our own `Submitting Patches <https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst>`_ document.
```

Comment:
I just want to say that this link works just fine for me.

Reply:
Groovy. When it was the (preferred) reference-by-name style it didn't; it seems that by accident or design .rst files at the top level aren't included in the `doc/` namespace by Sphinx, so this one had to be this sort of link.  The rendering isn't consistent, but at least it connects the two documents for now until they get merged together.